src/S.bs.js

```diff
@@ -126,30 +126,30 @@ function toReason(nestedLevelOpt, error) {
     case /* UnexpectedValue */2 :
         break;
     case /* TupleSize */3 :
-        return "Expected Tuple with " + reason.expected.toString() + " items, received " + reason.received.toString();
+        return "Expected Tuple with " + reason.expected.toString() + " items, received " + reason.received.toString() + "";
     case /* ExcessField */4 :
         return "Encountered disallowed excess key \"" + reason._0 + "\" on an object. Use Deprecated to ignore a specific field, or S.Record.strip to ignore excess keys completely";
     case /* InvalidUnion */5 :
-        var lineBreak = "\n" + " ".repeat((nestedLevel << 1));
+        var lineBreak = "\n" + " ".repeat((nestedLevel << 1)) + "";
         var array = reason._0.map(function (error) {
               var reason = toReason(nestedLevel + 1, error);
               var nonEmptyPath = error.path;
               var $$location = nonEmptyPath.length !== 0 ? "Failed at " + formatPath(nonEmptyPath) + ". " : "";
-              return "- " + $$location + reason;
+              return "- " + $$location + "" + reason + "";
             });
         var reasons = Array.from(new Set(array));
-        return "Invalid union with following errors" + lineBreak + reasons.join(lineBreak);
+        return "Invalid union with following errors" + lineBreak + "" + reasons.join(lineBreak) + "";
     
   }
-  return "Expected " + reason.expected + ", received " + reason.received;
+  return "Expected " + reason.expected + ", received " + reason.received + "";
```

Comment:
Started adding empty strings around values in template literal.

Reply:
Can't seem to repro:
```rescript
// tst.res
let template = (nestedLevel, reasons, foo) => {
  let lineBreak = `\n${" "->Js.String2.repeat(nestedLevel * 2)}`
  `Invalid union with following errors${lineBreak}${reasons->Js.Array2.joinWith(lineBreak)}`
}
```

gives

```js
// Generated by ReScript, PLEASE EDIT WITH CARE
'use strict';


function template(nestedLevel, reasons, foo) {
  var lineBreak = "\n" + " ".repeat((nestedLevel << 1));
  return "Invalid union with following errors" + lineBreak + reasons.join(lineBreak);
}

exports.template = template;
/* No side effect */
```

Would you open an issue with a self-contained repro?

Reply:
I've created an issue https://github.com/rescript-lang/rescript-compiler/issues/5521